docs/xostor/xostor.md

```diff
@@ -659,14 +659,25 @@ xe host-call-plugin host-uuid=<HOST_UUID> plugin=linstor-manager fn=addHost args
 ```
 For a short description, this command (re)creates a PBD, opens DRBD/LINSTOR ports, starts specific services and adds the node to the LINSTOR database.
 
-If you have storage devices to use on the host, a LINSTOR storage layer is not directly added to the corresponding node. You can verify the storage state like this:
+If you have storage devices to use on the host, a LINSTOR storage layer is not directly added to the corresponding node.
+You can follow the section below to add storage to this new node.
+
+### How to add storage on a new host
+
+On a existing host without storage, you can easily add storage by adding a new Linstor Storage Pool for the Linstor node.
```

Comment:
Too much redundancy :sweat_smile: Can we remove this sentence? The two steps are already clear, no?